### PR TITLE
[5.3] Graceful degradation for logging out when JavaScript is disabled

### PR DESCRIPTION
**Pull request by vlakoff**
Refs #14284 and [comments on commit 244bec95](https://github.com/laravel/framework/commit/244bec950c5e5af03f20fc9aca57b74a60e4935c).

ping @sileence, @tillkruss :)

### CODE REVIEW

File: src/Illuminate/Foundation/Auth/AuthenticatesUsers.php [hunk 1]
```diff
@@ -21,6 +21,16 @@ public function showLoginForm()
     }
 
     /**
+     * Show the application's logout form (used when JavaScript is disabled).
+     *
+     * @return \Illuminate\Http\Response
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Auth/AuthenticatesUsers.php]: This actually returns a view object doesn't it?

Comment (vlakoff) [inline src/Illuminate/Foundation/Auth/AuthenticatesUsers.php]: All the other methods that return views use this docblock...

**Comment by taylorotwell** [comment]
JavaScript is required. If a user doesn't want it to be required they can modify the stub.

**Comment by vlakoff** [comment]
I thought the benefit-cost ratio was worth it, but your pick :)

**Pull request closed without merging**